google/cloud/aiplatform/tensorboard/uploader.py

```diff
@@ -690,7 +693,7 @@ def __init__(
         max_request_size: int,
         tracker: upload_tracker.UploadTracker,
     ):
-        """Constructor for _ScalarBatchedRequestSender.
+        """Constructor for _NumericBatchedRequestSender.
```

Comment:
Is the name in this comment supposed to align with the name of class?

Reply:
I supposed so. Forgot to update the comment when renaming for the base class. Fixed.